### ISSUE TITLE
Data tables not keyboard accessible?

### ISSUE THREAD

**Issue by terribly** [wet-boew/wet-boew#9760]
I cannot see how the data tables are keyboard accessible. I do not see a means for getting to the first or any other row of data. Is this a known problem or am I missing something here?

**Comment by mercury64** [comment]
Keyboard accessible means that each time you press the 'tab' key, focus on the page moves to the next logical interactive item.

You could create/add tabindex like this example to get the functionality you want:
https://live.datatables.net/wodawore/1/edit
